packages/react/src/components/NumberInput/next/NumberInput.js

```diff
@@ -77,7 +75,6 @@ const NumberInput = React.forwardRef(function NumberInput(props, forwardRef) {
     [`${prefix}--number--light`]: light,
     [`${prefix}--number--nolabel`]: hideLabel,
     [`${prefix}--number--nosteppers`]: hideSteppers,
-    [`${prefix}--number--mobile`]: isMobile,
```

Comment:
Did we get the styles deleted for this one too?

Reply:
@tay1orjones I checked the v11 styles and they had already been removed. 👍🏽